gammapy/background/ring.py

```diff
@@ -51,21 +267,25 @@ class RingBackgroundEstimator(object):
 
     See Also
     --------
-    KernelBackgroundEstimator
+    KernelBackgroundEstimator, AdaptiveRingBackgroundEstimator
```

Comment:
This link to KernelBackgroundEstimator doesn't work:
http://docs.gammapy.org/en/latest/api/gammapy.background.RingBackgroundEstimator.html

Maybe you just have to write it out in full?
gammapy.detect.KernelBackgroundEstimator ?